### PR TITLE
fix: this set  default values to the content of .env instead of .env.…

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Overrides .env.development values with the content of .env

## Before
before-env

## After
after-env

### CODE REVIEW

File: webpack.prod.config.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+const dotenv = require('dotenv');
 const path = require('path');
 const { createConfig } = require('@edx/frontend-build');
 const CopyPlugin = require('copy-webpack-plugin');
 
 const config = createConfig('webpack-prod');
 
+// Overides development variables, this is required because this line
+// https://github.com/nelc/frontend-build/blob/open-release/palm.nelp/lib/plugins/paragon-webpack-plugin/ParagonWebpackPlugin.js#L20-L22
+// is always executed no matter the environment
```
Comment (johanseto) [inline webpack.prod.config.js:11]:
```suggestion
// is always executed no matter the environment ?
```
Is using webpack-prod config file so I i think is prod env, or what do you think of env?

**Review by johanseto — COMMENTED** [review]
LGTM, I though in an  override flag in frontend build, but I think is the same,

File: webpack.prod.config.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+const envConfig = dotenv.parse(fs.readFileSync('.env'));
+Object.keys(envConfig).forEach(k => {
+  process.env[k] = envConfig[k];
+});
+
```
Comment (johanseto) [inline webpack.prod.config.js:16]:
```suggestion
dotenv.config({
  path: path.resolve(process.cwd(), '.env'),
  override:  true,
});
```
I think that could do the same, 
inspired in this frontend build way 
https://github.com/openedx/frontend-build/blob/v12.8.42/config/webpack.prod.config.js#L28-L30
but with override
https://github.com/openedx/frontend-build/blob/v12.8.42/config/webpack.prod.config.js#L28-L30

You could tried...

Comment (andrey-canon) [reply webpack.prod.config.js:16]: the override feature was included in the 15.0.0 version we are in the  8.6

Comment (johanseto) [reply webpack.prod.config.js:16]: So the way you did it, is the way.

**Review by johanseto — APPROVED** [review]

File: webpack.prod.config.js [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
 
 const config = createConfig('webpack-prod');
 
+// The configuration is generated using the createConfig method from the frontend-build library,
+// this method preloads multiple files to generate the resulting configuration, including webpack.common.config.js,
+// https://github.com/nelc/frontend-build/blob/open-release/palm.nelp/config/webpack.common.config.js,
+// which includes ParagonWebpackPlugin. This plugin, in turn, retrieves its configuration from the .env.development file
+// https://github.com/nelc/frontend-build/blob/open-release/palm.nelp/lib/plugins/paragon-webpack-plugin/ParagonWebpackPlugin.js#L20-L22
+// Therefore, regardless of the configuration type, the plugin always utilizes data from .env.development.
+// The following code overrides this behavior in order to use the .env file.
```
Comment (johanseto) [inline webpack.prod.config.js:15]: now, it's really clear

**Pull request merged**